### ISSUE TITLE
Vote: what will Nature Zen be? Before February 28th 19:00

### ISSUE THREAD

**Issue by richelbilderbeek** [richelbilderbeek/djog_unos_2018#379]
**Is your feature request related to a problem? Please describe.**

Currently, we have a well-trained team that can solve any random Issue I create.

Now we need direction.

What will Nature Zen be? What is the goal of the game (again)? What will the complete ecosystem be like? Which must we do to finish the game before June?

**Describe the solution you'd like**

Vote by using this table:

Name|Vote yes/no
---|---
Jan|.
Joshua|.
Rafayel|.
Rob|.

**Describe alternatives you've considered**

None.

**Additional context**

The person that submitted the idea with most votes, will get the role of 'lead game design' and has the duty to create the Issues needed to finish the game. Sure, mediors and senior will still create Issues related to the code.

**Comment by Joshua260403** [comment]
The goal of the game should, as chosen before, to keep the zen. The zen is the balance in nature, a balance between life and death, carnivores and herbivores (and omnivores), plants and animals, or to be more clear: The zen is determined by the amount of agents relative to the amount of tiles. If there is under population the zen goes to one side, and when there is over population, the zen goes to the other. When the zen is too far from the center (perfectly balanced as all things should be), the game is over and the player lost. This is the basic gamemode. The player can also turn on sandbox mode (or different cheats). What we need to create is an ecosystem, filled with animals and plants and different landscapes (biomes). Certain animals can't live here, but other animals need to have access to that biome to survive. The game is pretty much endless (that's why we can load and save the game). Eventually (after completing the base game), we could release an update containing campaign mode, which isn't endless. In another update we could release an option for humans, who make the game harder by building on tiles to reduce the amount of food on it or hunt on animals so they dont die of naturals cause. But that is all for an update after finishing the base game.

So to summarize: We create a game where agents live in a world made of biomes, the player should avoid over- or under population by moving tiles with agents around to make sure agents have all the resources they need to survive.

**Comment by RafayelGardishyan** [comment]
Nature zen is af fight between the nature and a perfectionist, the human. The game was originally based to make a perfect world, where everywhere is harmony. The rules of Darwins theory work and the fittest will survive but what if only one animal type is over? Can the world survive? No!
Your task is to keep the world in balance with all the animals, plants and other creatures living in peace and full harmony. Make the world you would like to live in. Use your fantasy. Be God!

Nature Zen will change your life forever, untill an armageddon will destroy your world ;)

**Comment by robkruger** [comment]
The goal would be to keep the nature in balance, as the others said. You have to make sure a certain species won't take over and try not to get any species become extinct. But you have to make sure the tiles, the climates, are also balanced. For example, if you put a desert and an arctic tile next to each other, they will affect each other and their climates change. The arctic will melt and transform in water, and the desert will cool down and also transform. After some time in the game, the human species is also added, who will pollute the climate. You can stop that, either by moving them from any source of life or create natural disasters. The main goal is to influence the world so that all the species are balanced and there is a wide variation of climates.

**Comment by janderkkotlarski** [comment]
A more technical view on the game:

Game goal: Develop a self-sustaining eco-system from the starting configuration

Tiles are discrete habitats that can be directly manipulated by the player and agents are living organisms that cannot be directly manipulated by the player and can only be influenced by manipulating the tiles they reside on.

Tiles can be: Deleted
Duplicated
Merged
Moved
Rotated

All tiles should be merged into new tiles, no exception, makes the game less frustrating.
This means that some tile type mergings may not render a different tile type.

Agents automatically can: Move
Must be dragged along with the tiles they reside on
Cross tiles if possible
Grow
Replicate
Exhaust nutrients/energy
Die from exhaustion
Die naturally from old age
Die on incompatible tile types
Eat
Die from being eaten
Hinder each other's growth when in each other's vicinity
Kill each other when too close together

Death is a natural occurrence and not all death is detrimental, so not all deaths should result in an imbalance in score.
Like wise, while replication is natural, not all replication is beneficial, so not all replication should result in a balancing of the score.
The score should be based on the amount of different agent types present in the game, the amount of each agent type, the density of each agent type on the relevant environment tiles, the average health/exhaustion parameter of each agent type and how many agents die due to detrimental environments.

Once the game is too imbalanced, the game is over and should automatically go back to the start screen or auto-quit.

**Comment by janderkkotlarski** [comment]
Name|Vote yes/no
---|---
Jan|Yes
Joshua|Yes
Rafayel|No
Rob|No

**Comment by Rijk-van-Putten** [comment]
Name|Vote yes/no
---|---
Jan|Yes
Joshua|Yes
Rafayel|No
Rob|No

**Comment by robkruger** [comment]
Name|Vote yes/no
---|---
Jan| No
Joshua| No
Rafayel| Yes
Rob| Yes

**Comment by Joshua260403** [comment]
Name|Vote yes/no
---|---
Jan| N
Joshua| Y
Rafayel| N
Rob| N

**Comment by richelbilderbeek** [comment]
Name|Vote yes/no
---|---
Jan| Y
Joshua| Y
Rafayel| Y
Rob| Y

**Comment by JolienGay** [comment]
Name|Vote yes/no
---|---
Jan| Y
Joshua| Y
Rafayel| Y/N 
Rob| Y

Joshua: The updates idea sounds great!  

Rafayel: A little bit of 'N' for this reason:
I love the underworld idea, however, I think it would be more suitable in another type of game. From my perspective, it seems like you guys want to convey to players the importance of a healthy ecosystem for all creatures, including humans. Since this currently a relevant societal topic, I think this should remain the focus of the game, however cool it would be to have evil creatures attacking us ;).

**Comment by RafayelGardishyan** [comment]
Name|Vote yes/no
---|---
Jan|No
Joshua|Yes
Rafayel|Yes
Rob|No

**Comment by Enzo9** [comment]
Name|Vote yes/no
---|---
Jan|No
Joshua|Yes
Rafayel|Yes/no 
Rob| yes

**Comment by Joshua260403** [comment]
All 'yes/no' will be counted as a no

**Comment by Joshua260403** [comment]
I'll count the votes this evening
